index.d.ts

```diff
@@ -140,11 +139,15 @@ import './types/pixi/helpers/controlIcon';
 import './types/pixi/helpers/ray';
 import './types/pixi/helpers/ruler';
 
-import './types/pixi/placeableObjects/note';
 import './types/pixi/placeableObjects/placeableObject';
-import './types/pixi/placeableObjects/tile';
 import './types/pixi/placeableObjects/token';
 import './types/pixi/placeableObjects/wall';
+import './types/pixi/placeableObjects/drawing';
+import './types/pixi/placeableObjects/ambientLight';
+import './types/pixi/placeableObjects/note';
+import './types/pixi/placeableObjects/ambientSound';
+import './types/pixi/placeableObjects/measuredTemplate';
+import './types/pixi/placeableObjects/tile';
```

Comment:
Just to mention this early: Can you please sort those alphabetically when you are done?

Reply:
Yeah, I was sorting them in the same order they appear in foundry.js, but alphabetical works fine too